app/test/analyzer/handlers_test.dart

```diff
@@ -0,0 +1,178 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+library pub_dartlang_org.handlers_test;
+
+import 'dart:async';
+import 'dart:isolate';
+
+import 'package:gcloud/db.dart';
+import 'package:test/test.dart';
+
+import 'package:pub_dartlang_org/shared/analyzer_service.dart';
+
+import 'package:pub_dartlang_org/analyzer/backend.dart';
+import 'package:pub_dartlang_org/analyzer/models.dart';
+import 'package:pub_dartlang_org/shared/task_client.dart';
+import 'package:pub_dartlang_org/shared/task_scheduler.dart' show Task;
+
+import '../shared/handlers_test_utils.dart';
+import '../shared/utils.dart';
+
+import 'handlers_test_utils.dart';
+
+void main() {
+  group('handlers', () {
+    group('not found', () {
+      scopedTest('/xxx', () async {
+        expectNotFoundResponse(await issueGet('/xxx'));
```

Comment:
I think you need to either return the future from `expectNotFoundResponse` or `await` it, otherwise `package:test` will think the test was done without errors and mark it as `PASS` even if it fails later.

(also in the other tests)